app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,53 @@
   </head>
 
   <body>
+      <% if flash[:notice] %>
+      <div class="notification is-info global-notification">
+        <p class="notice"> <%= notice %> </p>
+      </div>
+      <% end %>
+      <% if flash[:alert] %>
+        <div class="notification is-danger global-notification">
+          <p class="alert"> <%= alert %> </p>
+        </div>
+      <% end %>
```

Comment:
There should be absolutely **NO logic in views**.  Views are only for displaying information that became available to it from its corresponding controller not generating it.

View-related logic can be encapsulated in View helpers (app/helpers/\*_helper.rb)

This [article ](https://medium.com/@acushla4real/rails-view-helpers-best-practices-guide-d7d7b6db3db4)would come in handy